client/app/toggleNavbar/toggleNavbar.js

```diff
@@ -1,9 +1,30 @@
-angular.module('parksAndEx.toggleNavbar', [])
-
-.controller('toggleNavbarController', function($scope) {
-  $scope.tab1 = false;
-  $scope.tab2 = false;
-  $scope.tab3 = false;
-  $scope.tab4 = false;
-  $scope.tab5 = false;
-});
+angular.module('parksAndEx.Tabstoggle', [])
+
+.controller('TabsCtrl', ['$scope', function ($scope) {
```

Comment:
This line can just be .controller('TabsCtrl', function($scope) {
And then on line 30 you would just need });